src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -25,6 +27,9 @@ public static void ClassInitialize(TestContext context)
 
             // Ensure that calculator is in standard mode
             page.NavigateToStandardCalculator();
+
+            // Ensure that calculator window is large enough to display the memory/history panel; a good size for most tests
+            page.MemoryPanel.ResizeWindowToDiplayMemoryLabel();
```

Comment:
Since the history panel is shown by default should you look for that or just for the pivot?

Reply:
Change in the next commit